zanshinsdk/client.py

```diff
@@ -199,92 +207,344 @@ def _request(self, method: str, path: str, params=None, body=None) -> httpx.Resp
+    def get_freshdesk_id(self) -> Dict:
+        """
+        Gets user freshdesk identifier.
+        <https://api.zanshin.tenchisecurity.com/#operation/getFreshdesk>
+        :return: a freshdesk unique identifier
+        """
+        return self._request("GET", "/me/freshdesk").json()
+
     def iter_organizations(self) -> Iterator[Dict]:
         """
-        Iterates over the organizations the API key owner has access to.
-        :return: an iterator over the organization objects
+        Iterates over organizations of current logged user.
+        <https://api.zanshin.tenchisecurity.com/#operation/getOrganizations>
+        :return: an iterator over the organizations objects
         """
         yield from self._request("GET", "/organizations").json()
 
-    def iter_scan_targets(self, organization_id: Union[UUID, str]) -> Iterator[Dict]:
+    def get_organization(self, organization_id: Union[UUID, str]) -> Dict:
         """
-        Iterates over the scan targets of an organization.
+        Gets an organization given its ID.
+        <https://api.zanshin.tenchisecurity.com/#operation/getOrganizationById>
         :param organization_id: the ID of the organization
-        : return: an iterator over the scan target objects
+        :return: a dict representing the organization detail
         """
-        yield from self._request("GET", f"/organizations/{validate_uuid(organization_id)}/scantargets").json()
+        return self._request("GET", f"/organizations/{validate_uuid(organization_id)}").json()
 
-    def _get_organization_alerts_page(self, organization_id: Union[UUID, str],
-                                      scan_target_ids: Optional[Iterable[Union[UUID, str]]] = None,
-                                      states: Optional[Iterable[AlertState]] = None,
-                                      severities: Optional[Iterable[AlertSeverity]] = None, page: int = 1,
-                                      page_size: int = 100) -> Dict:
+    def update_organization(self, organization_id: Union[UUID, str], name: Optional[str], picture: Optional[str], email: Optional[str]) -> Dict:
         """
-        Internal method to retrieve a single page of alerts from an organization.
+        Update organization given its ID.
+        <https://api.zanshin.tenchisecurity.com/#operation/editOrganizationById>
         :param organization_id: the ID of the organization
-        :param scan_target_ids: optional list of scan target IDs to list alerts from, defaults to all
-        :param states: optional list of states to filter returned alerts, defaults to all
-        :param severities: optional list of severities to filter returned alerts, defaults to all
-        :param page: page number, starting from 1
-        :param page_size: page size in number of alerts
-        :return:
+        :param name: the Name of the oranization
+        :param picture: the picture of the organization
+        :param email: the e-mail contact of the organization
```

Comment:
What exactly do you expect here in this parameter? A URL? The binary contents of an image? What are the supported formats?